DrFujiBot_Django/dashboard/admin_commands.py

```diff
@@ -350,9 +350,14 @@ def handle_latestquote(args):
     return output
 
 def handle_addquote(args):
-    quote_text = ' '.join(args)
-    quotee_setting = Setting.objects.filter(key='Quotee')[0]
-    quote_object = Quote(quote_text=quote_text, quotee=quotee_setting.value)
+    if args[0] == "-q":
+        quotee = args[1]
+        quote_text = ' '.join(args[2:])
+        quote_object = Quote(quote_text=quote_text, quotee=quotee)
+    else:
+        quote_text = ' '.join(args)
+        quotee_setting = Setting.objects.filter(key='Quotee')[0]
```

Comment:
You should probably use `Settings.objects.get(key='Quotee')` here instead of filter.